includes/wc-core-functions.php

```diff
@@ -1553,6 +1553,19 @@ function wc_uasort_comparison( $a, $b ) {
 	return ( $a < $b ) ? -1 : 1;
 }
 
+/**
+ * Sort values based on ascii, usefull for special chars in strings.
+ *
+ * @param string $a First value.
+ * @param string $b Second value.
+ * @return int
+ */
+function wc_ascii_uasort_comparison( $a, $b ) {
+	$aconv = iconv( 'UTF-8', 'ASCII//TRANSLIT', $a );
```

Comment:
I'm not sure `iconv` is available on all installs - we have function exists checks in other places. Is that neeed here too?